Tasks/DockerComposeV0/dockercompose.ts

```diff
@@ -57,7 +57,7 @@ if (nopIfNoDockerComposeFile && !tl.exist(dockerFile)) {
                 "combine configuration": "./dockercomposeconfig",
                 "run a docker compose command": "./dockercomposecommand"
             }[action]).run(connection, (pathToResult) => {
-                resultPaths += pathToResult;    
+                resultPaths += `${pathToResult}\n`;    
```

Comment:
When you use `\n` as delimiter, you won't be able to set the output variable correctly. It would render the `\n` in the logs and the agent would set the first one to be the value of the output variable

Reply:
It works exactly the same this way in DockerV2. I've also just tested it and everything seems to work fine:

I created a Bash task to log the output:
```
echo 'Output Docker Compose:'
echo "$(dockerCompose.DockerComposeOutput)"

echo
echo 'Cat stuff:'
echo "$(dockerCompose.DockerComposeOutput)"  | while read line; do echo $line && echo '' && cat $line && echo '' && echo '' && echo '' && echo '' && echo '' && echo ''; done
```

And these are the first few lines of the output:
![image](https://user-images.githubusercontent.com/2350015/79149423-6eda8500-7dc7-11ea-96eb-88b5d8d761c9.png)